dashboard.php

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+</head>
+<body>
+
+    <h2>Dashboard</h2><br/>
+    <h3>Please select section that you would like to edit:</h3><br/><br/>
```

Comment:
should use margin on this instead of BR. i wont ask you to do this project but for future ref